frontend/src/metabase/selectors/app.ts

```diff
@@ -69,9 +69,13 @@ export const getIsAppBarVisible = createSelector(
     embedOptions,
   ) => {
     const isFullscreen = hash.includes("fullscreen");
+    const allTopNavElementsHidden =
```

Comment:
[Following this new spec](https://www.notion.so/metabase/Improve-collection-breadcrumbs-with-embedding-parameters-51dd704168e344158b649415fb693960#0ef917ff09244d549fa3a269524c8378), if all top nav elements are hidden then we should hide the top nav too, since it'll serve no purpose.